Polyhedron/demo/Polyhedron/CMakeLists.txt

```diff
@@ -351,8 +351,9 @@ if(CGAL_Qt5_FOUND AND Qt5_FOUND)
   target_link_libraries(polyhedron_demo PUBLIC
     demo_framework point_dialog Qt5::Gui Qt5::OpenGL Qt5::Widgets Qt5::Script)
   if(LIBSSH_FOUND)
-    add_definitions(-DCGAL_USE_SSH)
+    target_compile_definitions(polyhedron_demo PUBLIC -DCGAL_USE_SSH)
```

Comment:
You can do the same for the `add_definitions(-DCGAL_USE_WEBSOCKETS)` a few lines below.

Reply:
I am surprised that the definition is `PUBLIC`. I do not think that all items and plugins of the demo really need that `-DCGAL_USE_SSH`.